torchvision/ops/boxes.py

```diff
@@ -382,7 +382,39 @@ def _box_diou_iou(boxes1: Tensor, boxes2: Tensor, eps: float = 1e-7) -> Tuple[Te
     # distance between boxes' centers squared.
     return iou - (centers_distance_squared / diagonal_distance_squared), iou
 
+def masks_to_boundaries(masks: torch.Tensor, dilation_ratio: float = 0.02) -> torch.Tensor:
```

Comment:
I guess it's OK to have the implementation in this file even though this isn't related to boxed. However, I don't think we should **expose** it here. I think we should just expose it in from the `torchvision.ops` namespace (otherwise the implementation will always have to stay in this file for BC, and that may lock us).

We probably just need to rename this to `_masks_to_boundaries` and the expose it in `torchvision.ops.__init__.py` like

```py
from .boxes import import _masks_to_boundaries as masks_to_boundaries
```

Any other suggestion @pmeier @vfdev-5 @oke-aditya ?

Reply:
> I guess it's OK to have the implementation in this file even though this isn't related to boxed. 

No strong opinion, but could we maybe also have a new `_masks.py` module or move it into the `misc.py` one?

:+1: for only exposing it in the `torchvision.ops` namespace.

Reply:
Tbh there is demand for mask_utils. Several of them, #4415 . Candidate utils like `convert_masks_format`, `paste_masks_in_images`, etc. Maybe it's time to create new files `mask_utils.py` and make future extensions possible?

Reply:
we can always create an `ops.mask*` namespace at any time. We should only do that when we know for sure we need it, i.e. when we start having 2+ mask utils. Alls ops are exposed in the `ops.` namespace anyway so there's no need to rush and create a file which will only have one single util in it ATM.

I'm OK with creating `_mask.py` as well (and we can rename it into `mask.py` later if we want to).

Reply:
> I'm OK with creating _mask.py as well (and we can rename it into mask.py later if we want to).

This sounds best solution! We can avoid the bloat inside this file as well as keep them private :smile: